src/uniqueArrayOf.js

```diff
@@ -1,20 +1,58 @@
-import { arrayOf } from 'prop-types';
+import { arrayOf, array } from 'prop-types';
 import and from './and';
 import uniqueArray from './uniqueArray';
 
 const unique = uniqueArray();
 
-export default function uniqueArrayOfTypeValidator(type, name = 'uniqueArrayOfType') {
+export default function uniqueArrayOfTypeValidator(type, ...rest) {
   if (typeof type !== 'function') {
     throw new TypeError('type must be a validator function');
   }
 
+  let mapper = rest.filter(element => typeof element === 'function');
```

Comment:
This currently doesn't care about order, so both of those should work. `mapper` will start out as an array with all of the functions in `rest`, and if there's more than 1 it will throw an error, otherwise it'll just take the single mapping function.

Right now, this allows [Function, String], [String, Function], [String], [Function], and []. Do you think making it more selective and only allowing those 3 orderings is better?